crates/bevy_render/src/color/mod.rs

```diff
@@ -683,6 +683,17 @@ impl Color {
         }
     }
 
+    /// Converts a `Color` to a `[u8; 4]` from sRGB colorspace
+    pub fn as_rgba_u8(&self) -> [u8; 4] {
+        let [r, g, b, a] = self.as_rgba_f32();
+        [
+            (r * 255.0) as u8,
+            (g * 255.0) as u8,
+            (b * 255.0) as u8,
+            (a * 255.0) as u8,
```

Comment:
Nit: Could be worth it to match `Color::rgba_u8` by using `u8::MAX`:
```suggestion
            (r * u8::MAX as f32) as u8,
            (g * u8::MAX as f32) as u8,
            (b * u8::MAX as f32) as u8,
            (a * u8::MAX as f32) as u8,
```